src/main/java/hudson/plugins/jira/pipeline/IssueUpdateStep.java

```diff
@@ -0,0 +1,110 @@
+package hudson.plugins.jira.pipeline;
+
+import com.google.inject.Inject;
+import hudson.Extension;
+import hudson.Util;
+import hudson.model.Result;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.plugins.jira.JiraSite;
+import hudson.plugins.jira.Messages;
+import org.apache.commons.lang.StringUtils;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepDescriptorImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepImpl;
+import org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution;
+import org.jenkinsci.plugins.workflow.steps.StepContextParameter;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+
+/**
+ * Step for updating jira issue with workflow migration
```

Comment:
Suggest to re-word, as I don't understand it - maybe let's stick to conform to JIRA terminology about workflow *transition* :-)
e.g. "Perform workflow transitions on the issues selected by JQL query"?